nss-day-cohort-41/tabloid-fullstack-shocking-secret-keepers
# Th recommended

# Description
Recommends posts from users that their subscribees are subscribed to. The top three posts in the post tab should show recommended posts.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Go log into a user and subscribe to a couple users and the people they are subscribed to should be showing up at the top of the post list. Make sure to add subscriptions to the other users otherwise nothing will happen. It should be randomized each time you travel to the page or refresh.
# Checklist:
